### PR TITLE
docs(jokes): add example code to test error boundaries

### PR DESCRIPTION
**Pull request by justinnoel**
This fixes a minor issue in the Jokes tutorial. In the [error boundaries section](https://remix.run/docs/en/v1/tutorials/jokes#unexpected-errors), it says:

> Ok great, with those in place, let's check what happens when there's an error. Go ahead and just add this to the default component, loader, or action of each of the routes. Here's what I get:

This indicates there should be some code sample for the user to add to the code. However, none is provided.

This PR simply adds some.

See also: https://github.com/remix-run/remix/pull/530#issuecomment-976873148

### CODE REVIEW

File: docs/tutorials/jokes.md [hunk 1]
```diff
@@ -4063,7 +4063,11 @@ export function ErrorBoundary() {
 
 </details>
 
-Ok great, with those in place, let's check what happens when there's an error. Go ahead and just add this to the default component, loader, or action of each of the routes. Here's what I get:
+Ok great, with those in place, let's check what happens when there's an error. Go ahead and just add this to the default component, loader, or action of each of the routes.
+
+`throw "Testing Error Boundary"`
```
Comment (kentcdodds) [inline docs/tutorials/jokes.md:4068]:
````suggestion
```ts
throw new Error("Testing Error Boundary")
```
````

**Review by kentcdodds — APPROVED** [review]
Good call. Thanks!
